nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/performingarts/PerformingArtsSubtypeEnum.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.model.instancetypes.artistic.performingarts;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum PerformingArtsSubtypeEnum {
+    THEATRICAL_PRODUCTION("TheatricalProduction"),
```

Comment:
Thinking about AWS SDK2, and its problems with additional labels, is there a reason to have additional labels? Is it to be consistent with the types produced by Jackson (i.e., class names)?